mysite/lti/views.py

```diff
@@ -80,20 +88,29 @@ def lti_redirect(request, course_id=None, unit_id=None):
     |  and/or UserSocialAuth.
     |  Finally login Django user and redirect to Course
 
-    :param course_id: course id from launch url
```

Comment:
> This view still takes course_id parameter, so why remove it from the docstring??

Done in https://github.com/raccoongang/socraticqs2/commit/f69d50f329fb42fdc2718bdb2369788203744139